appcircle_ios_objectivecswift_cache_push/1.0.2/component.yaml

```diff
@@ -3,8 +3,8 @@ buildPlatform: ObjectiveCSwift
 displayName: "Cache Push"
 description: "Uploads user selected files and folders to Appcircle cache."
 docsUrl: https://docs.appcircle.io/workflows/common-workflow-steps#cache-push
-requiredComponents: "appcircle_cocoapods"
-precedingComponents: "appcircle_ios_test,appcircle_ios_build_simulator,appcircle_ios_build_sign"
+requiredComponents: 
```

Comment:
Unfortunately `Cache Push` step is depended with `Git Clone`.
You can check this dependency => https://github.com/appcircleio/appcircle-cache-push-component/blob/ab0c2ac8510968b3b50af8a917075e409853fe0c/main.rb#L210
Can you made this like before:
```
requiredComponents: "appcircle_git_clone"
```